docs/swarm/swarm-tutorial/drain-node.md

```diff
@@ -97,7 +97,7 @@ task assignments for the `redis` service:
     9bg7cezvedmkgg6c8yzvbhwsd  redis.3       redis:3.0.6  worker2   Running        Running 4 minutes
     ```
 
-    The Swarm manager maintains the desired state by ending the task on a node
+    The swarm manager maintains the desired state by ending the task on a node
```

Comment:
Out of scope of this PR,  but I think this comment is backward:

The swarm manager maintains the desired state by starting a replica on a new node to replace the replica from the drained node.